grpc/src/test/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunctionBuilderTest.java

```diff
@@ -89,19 +89,24 @@ void sortExceptionHandler() {
                                  B1Exception.class);
 
         final GrpcExceptionHandlerFunction exceptionHandler = builder.build();
-        Status status = GrpcStatus.fromThrowable(exceptionHandler, ctx, new A3Exception(), new Metadata());
+        Status status = DefaultGrpcExceptionHandlerFunction.fromThrowable(exceptionHandler, ctx,
```

Comment:
We don't have to use `DefaultGrpcExceptionHandlerFunction` to call the `exceptionHandler`.
If `DefaultGrpcExceptionHandlerFunction` is necessary, `exceptionHandler.orElse(GrpcExceptionHandlerFunction.ofDefault()` can be used.

Reply:
Applied feedback. Thanks